Library/Formula/open-scene-graph.rb

```diff
@@ -33,8 +33,6 @@ class OpenSceneGraph < Formula
   depends_on "qt5" => :optional
   depends_on "qt" => :optional
 
-  # patch necessary to ensure support for gtkglext-quartz
-  # filed as an issue to the developers https://github.com/openscenegraph/osg/issues/34
   patch :DATA
```

Comment:
Can you add a link/explanation as to what the new patch is, where it's from, etc? Thanks.